rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -0,0 +1,896 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hsearch" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis for migration from Hibernate Search 4.x to Hibernate Search 5.x.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.4.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
+        </dependencies>
+        <sourceTechnology id="hibernate-search" versionRange="[4,)" />
+        <targetTechnology id="hibernate-search5" versionRange="[5,)" />
+        <targetTechnology id="eap7" versionRange="[7,)" />
+        <tag>reviewed-2016-04-12</tag>
+    </metadata>
+    <rules>
+        <rule id="hsearch-00000">
+            <when>
+                <javaclass references="org.hibernate.search.{*}" />
+            </when>
+            <perform>
+                <classification title="Hibernate Search usage" effort="0" severity="mandatory">
```

Comment:
What is this used for? I see we have `<classification>`s in `eap6[-xml].windup.xml`, but they mostly seem like they should be `<hint>`s instead. Other than that, we have no `<classification>`s anywhere.